### PR TITLE
Added the auth paths

### PR DESCRIPTION
**Pull request by ScoobyDubeyDo**
- added auth using Mockbee
- sign-in page and functionality
- sign up page  and functionality
- added es lint > please ignore `package-lock.json`, `package.json`,`AuthController.js`,`authUtils.js` files

### CODE REVIEW

File: src/utils/userSignin.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                    password: passwordRef.current.value,
+                });
+                if (res.status === 200) {
+                    setCurrentUser({
+                        encodedToken: res.data.encodedToken,
+                        ...res.data.foundUser,
+                    });
+                    // localStorage.setItem("token", res.data.encodedToken);
+                    navigate("/");
+                    console.log(res);
```
Comment (shobit1337) [inline src/utils/userSignin.js:27]: Remove these console.log messages before pushing.

File: src/utils/userSignup.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                    email: emailRef.current.value,
+                    password: passwordRef.current.value,
+                    name: nameRef.current.value,
+                });
+                if (res.status === 201) {
+                    setCurrentUser({
+                        encodedToken: res.data.encodedToken,
+                        ...res.data.createdUser,
+                    });
+                    // localStorage.setItem("token", res.data.encodedToken);
```
Comment (shobit1337) [inline src/utils/userSignup.js:27]: remove unnecessary commented code.

**Review by shobit1337 — APPROVED** [review]
Your code Looks clean and very nicely written. Good work @aman-can 

and really liked your jwt_secret value 😂

**Review by muskaanshah — APPROVED** [review]
Looks good to merge

**Review by mohammadkhizerkhan — APPROVED** [review]
Everything looks good to me especially JWT KEY

**Pull request merged**